roles/cifmw_cephadm/tasks/post.yml

```diff
@@ -63,7 +63,15 @@
       vars:
         ceph_command: "radosgw-admin"
 
+    - name: Check if Ceph user exists
+      become: true
```

Comment:
Is become needed?

Reply:
yeah, behind the scenes the ceph client needs root access, but I didn't add it at block level because we don't need it in all tasks